monitor/service/api/__init__.py

```diff
@@ -14,8 +14,6 @@
 # limitations under the License.
 
 import ConfigParser
-import os
```

Comment:
This is a unused import? If yes, please comment this modification in the description of this PR. This issue repeats itself through all file.